src/main/java/org/opentripplanner/updater/transportation_network_company/NoApiTransportationNetworkCompanyDataSource.java

```diff
@@ -0,0 +1,89 @@
+package org.opentripplanner.updater.transportation_network_company;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.opentripplanner.routing.transportation_network_company.ArrivalTime;
+import org.opentripplanner.routing.transportation_network_company.RideEstimate;
+import org.opentripplanner.routing.transportation_network_company.TransportationNetworkCompany;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This data source is to model a transportation network company for which no API exists to calculate real-time arrival
+ * estimates or ride estimates. The config for this updater can include a default value for the estimated arrival time
+ * which will always be provided when estimates for the arrival time are desired.
```

Comment:
JavaDoc should probably mention that this is primarily for testing.

Reply:
It's not primarily for testing.